iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -331,6 +331,23 @@ static void splitParallelAndReductionTiles(
   }
 }
 
+static void setAlwaysVectorizeSizes(linalg::LinalgOp op,
+                                    SmallVectorImpl<int64_t> &parallelSizes,
+                                    SmallVectorImpl<int64_t> &reductionSizes) {
+  Optional<SmallVector<int64_t, 4>> staticLoopRanges = op.getStaticLoopRanges();
+  for (auto en :
+       llvm::enumerate(llvm::zip(*staticLoopRanges, op.iterator_types()))) {
```

Comment:
Sorry, I'm a bit confused. I thought this would se tile sizes to 1 for dynamic dims but it's doing the opposite? Could you please add some doc to the method? That would be very helpful.

Reply:
hmm, the implementation is setting tile sizes to 1 for dynamic dims. If it is not dynamic dim, it will run into `continue` statement. If it is dynamic dim, the tile size will be set to 1.

Reply:
Sorry, I missed the `!` :). Thanks for clarifying! In any case, I think it would be good to add a small doc comment to the method. It's not easy to infer that `setAlwaysVectorizeSizes` is only about dynamic dims.

Reply:
oops, it's auto-merged. Good point, I'll send out a PR to add the comment!